app/views/layouts/application.html.erb

```diff
@@ -21,6 +21,8 @@
         <%= @hero_html.html_safe %>
       </div>
     <% end %>
+    
+    <%= render "layouts/asset_reconciliation" %>
```

Comment:
👍  I'm glad we're pulling this out of the main layout into a specific partial, a great change